tools/check_for_upgrade.sh

```diff
@@ -29,9 +29,9 @@ fi
 # Cancel upgrade if git is unavailable on the system
 whence git >/dev/null || return 0
 
-if [ -f ~/.zsh-update ]
+if [ -f ${ZSH_CACHE_DIR}.zsh-update ]
```

Comment:
Hi, thank you for the suggestion. I mistakenly thought it would cause problem to use a `/` here since the variable `$ZSH_CACHE_DIR` in `oh-my-zsh.sh` also contains a `/`. It's fixed now.